src/main.cpp

```diff
@@ -220,7 +239,7 @@ Hitable *simple_light()
 
 Hitable *cornell_box()
 {
-    Hitable **list = new Hitable*[7];
+    Hitable **list = new Hitable*[1000];
```

Comment:
Just to be safe: this way we don't have to worry about always updating this fkng value.